pkg/data/components/SymbolicatorProcessor.yaml

```diff
@@ -0,0 +1,60 @@
+kind: SymbolicatorProcessor
+name: Symbolicator Processor
+style: processor
+type: base
+status: development
+version: v0.1.0
+summary: A processor that will symbolicate JavaScript stack traces using source maps.
+description: |
+  This processor is used to symbolicate JavaScript stack traces using source maps.
+tags:
+  - category:processor
+  - service:collector
+  - signal:OtelTraces
+ports:
+  - name: TracesIn
+    direction: input
+    type: OTelTraces
+  - name: TracesOut
+    direction: output
+    type: OTelTraces
+properties:
+  - name: SourceMapStore
+    type: string
+    subtype: oneof(file_store, s3_store, gcs_store)
+    validations:
+      - oneof(file_store, s3_store, gcs_store)
+  - name: Path
+    type: string
+  - name: Bucket
+    type: string
+  - name: Region
+    type: string
+  - name: Prefix
+    type: string
+templates:
+  - kind: collector_config
+    name: otel_logdedup
+    format: collector
+    meta:
+      componentSection: processors
+      signalTypes: [traces]
+      collectorComponentName: symbolicator
+    data:
+      - key: "{{ .ComponentName }}.source_map_store"
+        value: "{{ .HProps.SourceMapStore }}"
+      # path is only required for file_store
+      - key: "{{ .ComponentName }}.path"
```

Comment:
Path is nested under `local_source_maps`: https://github.com/honeycombio/opentelemetry-collector-symbolicator/blob/4bf2dbdd368206a2a6bb118e6807848f3ca77984/symbolicatorprocessor/config.go#L63C74-L63C91